### PR TITLE
refactor(devops): Improve WASM source in tests

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Make tests more reliable instead of relying in relative paths.

# Changes

* Change the wasm paths to be relative to the cargo workspace instead of relative to the test file.
* Set the env vars in `test.backend.sh` accordingly.

# Tests

Running `test.backend.sh` with different wasm file names works.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@bitdivine please review

File: src/signer/tests/it/utils/pocketic.rs [hunk 1]
```diff
@@ -82,20 +96,36 @@ impl BackendBuilder {
     /// To override, please use `with_cycles()`.
     pub const DEFAULT_CYCLES: u128 = 2_000_000_000_000;
     /// The default Wasm file to deploy:
-    /// - If the environment variable `BACKEND_WASM_PATH` is set, it will use that path.
-    /// - Otherwise, it will use the `BACKEND_WASM` constant.
+    /// - If the environment variable `SIGNER_CANISTER_WASM_FILE` is set, it will use that path.
```
Comment (bitdivine) [inline src/signer/tests/it/utils/pocketic.rs:99]: Is this the env var set by dfx?  I can't check the docs right now as this is down: https://internetcomputer.org/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-envars

**Review by bitdivine — APPROVED** [review]
LGTM.  Thanks!

File: src/signer/tests/it/utils/pocketic.rs (hunk 1 above)
Comment (lmuntaner) [reply src/signer/tests/it/utils/pocketic.rs:99]: No, this is set in the `test.backend.sh`. The test doesn't use `dfx`, so not sure about relying on it.

**Pull request merged**